server/src/main/java/com/google/coffeehouse/common/Person.java

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.common;
+
+import com.google.coffeehouse.util.IdentifierGenerator;
+import com.google.coffeehouse.util.UuidWrapper;
+import java.util.Optional; 
+
+/**
+ * Person is the class than encapsulates the information associated with a user who has an account.
+ * 
+ * <p>It implementes the {@link Saveable} interface because it will be able to save itself
+ * in the database.
+ */
+public class Person implements Saveable {
+  private String nickname;
+  private String email;
+  private String pronouns;
+  private String userId;
+
+  private Person(Builder builder) {
+    this.nickname = builder.nickname;
+    this.email = builder.email;
+    this.pronouns = builder.pronouns;
+    this.userId = builder.idGenerator.generateId();
+  }
+
+  public String getNickname() {
+    return nickname;
+  }
+
+  public String getEmail() {
+    return email;
+  }
+
+  /** Returns an optional because the pronouns may, or may not, exist. */
+  public Optional<String> getPronouns() {
+    return pronouns == null ? Optional.empty() : Optional.of(pronouns);
+  }
+
+  public String getUserId() {
+    return userId;
+  }
+
+  public void setPronouns(String pronouns) {
+    this.pronouns = pronouns;
+  }
+
+  public void setEmail(String email) {
+    this.email = email;
+  }
+
+  public void setNickname(String nickname) {
+    this.nickname = nickname;
+  }
+
+  /**
+   * Starts the building process of a new Person object
```

Comment:
A few nits here:
- End the first line with a period
- The param comments don't add much. As "email" and "nickname" are pretty self-explanatory, you can just include the name.
- The return comment could be rephrased: "that was instantiated by this method" is obvious.

Reply:
Thank you, I have removed the unneeded parts.